### ISSUE TITLE
Where is class?

### ISSUE THREAD

**Issue by ryanmilvenan** [ucdd2-sp15/mailinglist#2]
This may seem like an odd question. Where is this class taking place?

**Comment by asassu** [comment]
Since we haven't heard otherwise, I assume the 2nd floor lounge in the ATLAS building, since the website says that's our default meeting location. It's the area right outside of the stairwell, not in a room.

**Comment by doubleshow** [comment]
That's correct. DLC 170 2nd floor lounge, unless otherwise noted when we bring a hackathon to an off-site location.
